cmd/keycmd/transfer.go

```diff
@@ -572,6 +586,29 @@ func interEvmSend(
 		destinationAddr,
 		amountInt,
 	)
+	if err != nil {
+		return err
+	}
+
+	chainName, err := contract.GetBlockchainDesc(senderChain)
+	if err != nil {
+		return err
+	}
+	ux.Logger.PrintToUser("%s Paid fee: %.9f",
+		chainName,
+		calculateEvmFee(receipt.GasUsed, receipt.EffectiveGasPrice))
```

Comment:
worth its own func? seeing the same logic is being called below

Reply:
i meant the `GetBlockchainDesc->PrintToUsers->calculateEvmFee` call sequence. no strong opinions here actually, just out of sake of aesthetics

Reply:
There are one more occurrence of similar block, I think it's not worth to extract